Traceback (most recent call last):
  File "/home/pi/FaceIdentification/train.py", line 44, in <module>
    recognizer.train(x_train,np.array(y_labels))
cv2.error: OpenCV(3.4.4) /home/pi/packaging/opencv-python/opencv_contrib/modules/face/src/lbph_faces.cpp:362: error: (-210:Unsupported format or combination of formats) Empty training data was given. You'll need more than one sample to learn a model. in function 'train'
